src/v/redpanda/admin/api-doc/debug_bundle.def.json

```diff
@@ -0,0 +1,117 @@
+"scram_authn": {
+  "type": "object",
+  "required": ["mechanism", "username", "password"],
+  "properties": {
+    "mechanism": {
+      "description": "Scram mechanism",
+      "type": "string"
+    },
+    "username": {
+      "description": "username used by RPK to authenticate against Kafka and Admin API",
+      "type": "string"
+    },
+    "password": {
+      "description": "password used by RPK to authenticate against Kafka and Admin API",
+      "type": "string"
+    }
+  }
+},
+"bundle_start_config_params": {
+  "type": "object",
+  "required": [],
+  "properties": {
+    "authentication": {
+      "description": "Authentication object",
+      "type": "scram_authn"
+    },
+    "controller_logs_size_limit_bytes": {
+      "type": "int"
+    },
+    "cpu_profiler_wait_seconds": {
+      "type": "int"
+    },
+    "logs_since": {
+      "description": "See “rpk debug bundle --help” about the format.  Expecting same format here.",
+      "type": "string"
+    },
+    "logs_size_limit_bytes": {
+      "type": "int"
+    },
+    "logs_until": {
+      "description": "See “rpk debug bundle --help” about the format.  Expecting same format here.",
+      "type": "string"
+    },
+    "metrics_interval_seconds": {
+      "type": "int"
+    },
+    "partition": {
+      "type": "string"
+    }
+  }
+},
+"bundle_start_config": {
```

Comment:
Sorry @BenPope @michael-redpanda - I saw this [thread](https://github.com/redpanda-data/redpanda/pull/23366#discussion_r1765425721) but wasn't sure if that was implemented or if that might have answered my question. The schema currently indicates that the [request body](https://github.com/redpanda-data/redpanda/pull/23366/files#diff-e2ced4f335d34fdc97bbd85299c4800ae900ae1a3f4a94f8921a47ad5ec9e719R15) must be a `config` object and that object requires a `job_id`. Where does one find the job UUID - is it not the job that will be kicked off by calling the endpoint?

Reply:
> Sorry @BenPope @michael-redpanda - I saw this [thread](https://github.com/redpanda-data/redpanda/pull/23366#discussion_r1765425721) but wasn't sure if that was implemented or if that might have answered my question. The schema currently indicates that the [request body](https://github.com/redpanda-data/redpanda/pull/23366/files#diff-e2ced4f335d34fdc97bbd85299c4800ae900ae1a3f4a94f8921a47ad5ec9e719R15) must be a `config` object and that object requires a `job_id`. Where does one find the job UUID - is it not the job that will be kicked off by calling the endpoint? 

I think the caller must provide a v4 UUID.

Reply:
@BenPope do they call a separate endpoint or run an rpk command to retrieve that uuid? (The doc for [`rpk debug`](https://docs.redpanda.com/current/reference/rpk/rpk-debug/rpk-debug/) doesn't have much detail and [`rpk debug bundle`](https://docs.redpanda.com/current/reference/rpk/rpk-debug/rpk-debug-bundle/) doesn't mention a job UUID either.

Maybe related - we only have a handful of the [debug endpoints](https://docs.redpanda.com/api/admin-api/#tag--Debugging) officially documented. I have an [old backlog issue](https://github.com/redpanda-data/documentation-private/issues/1922) to doc debug bundle but I don't know if the PRD and related issues/PRs are out of date. In any case there's a good number of debug endpoints in the [spec](https://github.com/redpanda-data/redpanda/blob/dev/src/v/redpanda/admin/api-doc/debug.json) that we don't have in the docs... Perhaps I should sync with you or @michael-redpanda about it?

Reply:
It's on the caller to generate a valid UUID which will become associated with the created process.

Reply:
@michael-redpanda I see, thanks!